locksmith/src/utils/middlewares/error.ts

```diff
@@ -0,0 +1,20 @@
+import { ZodError } from 'zod'
+import { ErrorRequestHandler } from 'express'
+
+export const errorHandler: ErrorRequestHandler = (
```

Comment:
Default Error middleware which we can use to handle all generic errors in our format.